<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFRDRTableHistogramDialog</class>
 <widget class="QDialog" name="QFRDRTableHistogramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculate Histogram</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>input column:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbInput"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>bins:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="spinBins">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkAutoBinWidth">
         <property name="text">
          <string>automatic bin width:</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFDoubleEdit" name="edtBinWidth">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>range:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="chkRangeMin">
         <property name="text">
          <string>min:</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFDoubleEdit" name="edtRangeMin">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>  ...  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkRangeMax">
         <property name="text">
          <string>max:</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFDoubleEdit" name="edtRangeMax">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>output:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="cmbOutput"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labStatistics">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="chkNormalized">
       <property name="text">
        <string>normalized</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFPlotter" name="plotter" native="true"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFPlotter</class>
   <extends>QWidget</extends>
   <header>qfplotter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QFDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qfdoubleedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QFRDRTableHistogramDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QFRDRTableHistogramDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkRangeMin</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtRangeMin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkRangeMax</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtRangeMax</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAutoBinWidth</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtBinWidth</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAutoBinWidth</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBins</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbInput</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QFRDRTableHistogramDialog</receiver>
   <slot>updateHistograms()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbOutput</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QFRDRTableHistogramDialog</receiver>
   <slot>updateHistograms()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkNormalized</sender>
   <signal>toggled(bool)</signal>
   <receiver>QFRDRTableHistogramDialog</receiver>
   <slot>updateHistograms()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateHistograms()</slot>
 </slots>
</ui>
